ccd-definition/AuthorisationCaseField/AuthorisationCaseField-SDO-nonprod.json

```diff
@@ -259,6 +259,58 @@
       }
     ]
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "disposalHearingMethod",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-admin",
+          "caseworker-civil-solicitor"
+        ],
+        "CRUD": "CRU"
+      }
+    ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "inPerson",
```

Comment:
Given this will exist on small claims and fast track, I'd recommend calling this something like `DisposalHearingMethodInPerson` to avoid confusion.

Similarly for the other options below

Reply:
Done